src/indev/lv_indev.c

```diff
@@ -844,10 +844,6 @@ static void indev_encoder_proc(lv_indev_t * i, lv_indev_data_t * data)
         data->enc_diff = 0;
     }
 
-    /*Refresh the focused object. It might change due to lv_group_focus_prev/next*/
```

Comment:
maybe can delete the previous one and maintain this comment

Reply:
See the original comment in https://github.com/lvgl/lvgl/pull/4930, we decided to use the first one to keep functionality unchanged.